pkg/processor/test/offline/suite_test.go

```diff
@@ -30,19 +31,28 @@ type offlineTestSuite struct { // nolint
 	httpsuite.TestSuite
 }
 
+func (suite *offlineTestSuite) SetupTest() {
+	suite.TestSuite.SetupTest()
+
+	// since we build offline, force docker build command to run with --network none
+	err := os.Setenv("NUCLIO_DOCKER_BUILD_NETWORK", "none")
+	suite.Require().NoError(err)
+}
+
 func (suite *offlineTestSuite) TestGolang() {
-	createFunctionOptions := suite.GetDeployOptions("withvendor",
-		path.Join(suite.GetTestFunctionsDir(), "golang", "with-vendor"))
+	createFunctionOptions := suite.GetDeployOptions("withmodules",
```

Comment:
TODO add test (and make work) where the nuclio function given is with vendor and single file (without any)